Flipper/Packages/UI/Sources/Archive/CardView/Name/CardNameInfoView.swift

```diff
@@ -26,9 +28,14 @@ extension CardView {
                             .resizable()
                             .renderingMode(.template)
                             .frame(width: 24, height: 24)
-                            .foregroundColor(.primary)
+                            .foregroundColor(
```

Comment:
Would be better to extract the button to it's own view and use isEnabled to choose the color